### PR TITLE
Updated docs to reflect we support python 3.7

### LINKED ISSUE #189: Investigate support for earlier python versions
Would be nice to go back to python 3.7 due to collab support.

### PR DESCRIPTION
**Pull request by sjrl**
## Description

Fixes #189. Adds to the docs that we support Python 3.7

### CODE REVIEW

**Comment by sjrl** [comment]
I also removed `docs/gettingstarted.md` because it was out of date and is it not used by the docs

**Comment by wuhu** [comment]
How about adding a note saying that >=3.8 is preferred due to the slowness?

File: README.md [hunk 1]
```diff
@@ -31,7 +31,10 @@ Full documentation here: https://lf1-io.github.io/padl/
 pip install padl
 ```
 
-PADL currently supports python **3.8** and **3.9**.
+PADL currently supports python **3.7**, **3.8** and **3.9**.
+
+Python version >= **3.8** is preferred because creating and loading transforms (**not** execution) 
+can be slower in **3.7**.
```
Comment (sjrl) [inline README.md:37]: Is this alright @wuhu ?

Comment (wuhu) [reply README.md:37]: :+1:

**Pull request merged**